QualityControl/docs/Architecture.md

```diff
@@ -0,0 +1,26 @@
+# Project Architecture
+This document explains the architecture of the project, detailing the main components and their interactions.
+
+## Overview
+The project is a web-based graphical user interface (GUI) for O<sup>2</sup> Quality Control. It consists of a frontend built with Mithril.js and a backend built with Node.js and Express.js. The backend interacts with the Calibration and Conditions Database (CCDB) to retrieve and manage quality control data.
+
+## Components
+### 1. Frontend
+The frontend follows a structured approach with models and services to manage the application's state and interactions. `Models` represent the application state and actions. `Services` contain the business logic and interact with the backend to fetch and manage data. They make API calls and process the responses.
+
+- Framework: [Mithrill.js](https://mithril.js.org/)
```

Comment:
As we are using a custom version of Mithril, I think it might be better here to redirect to the framework tutorials and documentations. Some examples is:
- https://github.com/AliceO2Group/WebUi/blob/dev/Framework/docs/guide/template-engine.md
- https://github.com/AliceO2Group/WebUi/blob/dev/Framework/docs/guide/components.md